packages/components/data-table/README.md

```diff
@@ -70,10 +70,9 @@ return <DataTable rows={rows} columns={columns} />;
 | `label`                | `string`  | ✅       | -       | The label of the column that will be shown on the column header.                                                                                                                                                                                                                                                                                                                                                                                                      |
 | `align`                | `string`  | -        | -       | Use this to override the table's own `cellAlignment` prop for this specific column. Can be one of [`left`, `center`, `right`].                                                                                                                                                                                                                                                                                                                                        |
 | `width`                | `string`  | -        | `auto`  | Sets a `width` for this column. Accepts the same values as the ones specified for individual [grid-template-columns](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns). Some common useful examples are `minmax` pairs (e.g. `minmax(200px, 400px)`), combinations of fraction values (`1fr`/`2fr`/etc), or fixed values such as `200px`. By default, the column grows according to the content and respecting the total table available width. |
-| `onClick`              | `func`    | -        | -       | Function to make the cells of this column clickable by the user. Keep in mind that if used on a row that implements `onRowClick`, you might want to set the column property `shouldIgnoreRowClick` to ignore it for this column, otherwise, upon click this function will be called first and then `onRowClick` immediately afterwards.                                                                                                                               |
```

Comment:
No more onClick?

Reply:
Not for individual columns. This was a Design decision, to prevent wrong usages.